app/test/process_proposal_test.go

```diff
@@ -349,26 +328,6 @@ func TestProcessProposal(t *testing.T) {
 			appVersion:     v3.Version,
 			expectedResult: abci.ResponseProcessProposal_REJECT,
 		},
-		{
-			name:  "blob txs larger than configured max tx size",
-			input: validData(),
-			mutator: func(d *tmproto.Data) {
-				d.Txs = append(d.Txs, largeBlobTxs...)
-				d.Hash = calculateNewDataHash(t, d.Txs)
-			},
-			appVersion:     appconsts.LatestVersion,
-			expectedResult: abci.ResponseProcessProposal_REJECT,
```

Comment:
[blocking]

wait, isn't the problem that this doesn't occur now, and chaning it is consensus breaking? Is this just getting rejected for the reason we think it is?

Reply:
cool x-post https://github.com/celestiaorg/celestia-app/pull/4084#discussion_r1876539901

> changing in process proposal is consensus breaking

yeah this is just why I was confused we were testing things there